packages/gestalt/src/Checkbox.css

```diff
@@ -35,13 +37,13 @@
 }
 
 .checkSm {
-  border-radius: 3px;
+  border-radius: 4px;
```

Comment:
@AlbertCarreras Instead, can we compose from `rounding1`?

```
composes: rounding1 from "./Borders.css";
```

Reply:
DONE